cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -19,19 +24,23 @@ describe('Topology view', () => {
     ).as('editInstance');
   });
 
-  it('render the Topology page', () => {
+  after(() => {
+    user?.id && cy.deleteAwxUser(user, { failOnStatusCode: false });
+  });
+
+  it.skip('render the Topology page', () => {
     cy.navigateTo('awx', 'topology-view');
     cy.verifyPageTitle('Topology View');
   });
 
-  it('refresh the Topology page', () => {
+  it.skip('refresh the Topology page', () => {
     cy.navigateTo('awx', 'topology-view');
     cy.verifyPageTitle('Topology View');
     cy.get('#refresh').click();
     cy.verifyPageTitle('Topology View');
   });
 
-  it('show sidebar when node is clicked', () => {
+  it.skip('show sidebar when node is clicked', () => {
```

Comment:
I see that you have skipped this test- It looks like it would be a better component test. Perhaps it can be moved to the component test file for this feature, and deleted from this file?